taskfile/common.yml

```diff
@@ -37,11 +36,24 @@ tasks:
   tag:
     requires:
       vars: [ STAGE ]
+    vars:
+      TAG:
+        sh: TZ=ETC/UTC date +%Y-%m-%dT%H-%M-%SZ-{{.STAGE}}
     cmds:
       - task: next_upgrade
       - task: _tag
-        vars: { STAGE: "{{.STAGE}}" }
-
+        vars: { STAGE: "{{.STAGE}}", TAG : "{{.TAG}}" }
+  
+  lib-tag:
+    requires:
+      vars: [ STAGE ]
+    vars:
+      TAG:
+        sh: echo "v1".`TZ=ETC/UTC date +%Y%m%d`.`TZ=ETC/UTC date +%-H%M`
```

Comment:
Can't we var the v1 (and have it as default value) ?

Reply:
@erwanncloarec we've replaced the new method by a flag in the already existing `tag` method.